index.js

```diff
@@ -1,25 +1,25 @@
 "use strict";
 
-var Botkit = require('botkit')
-    , giphy = require('./giphy')
-    , joinAndCommas = require('./joinAndComma')
-    , Promise = require('bluebird')
-    , readFile = Promise.promisify(require("fs").readFile)
-    , emoji = "boom"
-    , react = require('./react')
-    , noop = ()=>undefined
-    , PERSIST_BINGO_WORDS = process.env.PERSIST_BINGO_WORDS && process.env.PERSIST_BINGO_WORDS != 0
-    , botUid
-    , words = []
-    , foundWords = []
-    , persistence = require('./persistence')
-    , bot
-    , controller = Botkit.slackbot({
+var Botkit = require('botkit'),
+    giphy = require('./giphy'),
+    joinAndCommas = require('./joinAndComma'),
+    Promise = require('bluebird'),
+    readFile = Promise.promisify(require("fs").readFile),
+    emoji = "boom",
+    react = require('./react'),
+    insultgen = require('insultgenerator'),
+    noop = () => undefined,
+    PERSIST_BINGO_WORDS = process.env.PERSIST_BINGO_WORDS && process.env.PERSIST_BINGO_WORDS != 0,
+    botUid, words = [],
+    foundWords = [],
+    persistence = require('./persistence'),
+    bot, controller = Botkit.slackbot({
         logLevel: 3
-        //debug: true
-        //include "log: false" to disable logging
-        //or a "logLevel" integer from 0 to 7 to adjust logging verbosity
-    })
+            //debug: true
+            //include "log: false" to disable logging
+            //or a "logLevel" integer from 0 to 7 to adjust logging verbosity
+    }),
+    sentenceMatcher = new RegExp(`((?:[A-Z]\.|[^\.!?])+)[\.!?]`, "g");
```

Comment:
It will accept also lower case letters in the beginning as this RegEx is not acting case sensitive.